repeat/p5js/hidden-circles/index.js

```diff
@@ -0,0 +1,69 @@
+var dis = 20;
+
+function setup() {
+  createCanvas(500, 500);
```

Comment:
the canvas needs to be assigned see https://github.com/fabianmoronzirfas/gestalten-in-code/wiki/Contributing-P5js-Sketches#code